magento/ece-tools
# MCLOUD-5637: Update all error codes in ece-tools

### Description
https://jira.corp.magento.com/browse/MCLOUD-5637


### Fixed Issues (if relevant)

https://jira.corp.magento.com/browse/MCLOUD-5637

### Manual testing scenarios

1. Deploy Magento with the wrong configuration in `.magento.env.yaml`
```yaml
stage:
    deploy:
        SCD_THREADS: "asd"
```
2. Check that the deployment process failed with error code different than 1.
```
[2020-05-22 14:45:01] ERROR: [10] Fix configuration with given suggestions:
        - Environment configuration is not valid. Correct the following items in your .magento.env.yaml file:
          The SCD_THREADS variable contains an invalid value of type string. Use the following type: integer.
````
3. Fix `.magento.env.yaml`  and add some wrong configuration through UI. For example 
`DATABASE_CONFIGURATION` => `{'key':'value'}`
4. Check that the deployment process finished with error code different than 1.
```
    [2020-05-22 13:52:57] ERROR: [109] Fix configuration with given suggestions:
    - Variable DATABASE_CONFIGURATION is not configured properly
      At least host, dbname, username and password options must be configured for default connection  
```
### Contribution checklist
